adapter/src/logger.ts

```diff
@@ -75,15 +80,28 @@ export class Logger {
 		}
 	}
 
+	private _formatTimestamp(): string {
+		let d = new Date();
+		let millisecondString = String("000" + d.getMilliseconds()).slice(-3);
+		return d.toLocaleDateString() + ' ' + d.toLocaleTimeString() + '.' + millisecondString;
+	}
+
 	/**
 	 * Set the logger's minimum level to log in the console, and whether to log to the file. Log messages are queued before this is
 	 * called the first time, because minLogLevel defaults to Warn.
 	 */
-	setup(consoleMinLogLevel: LogLevel, _logFilePath?: string|boolean): void {
+	setup(consoleMinLogLevel: LogLevel, _logFilePath?: string|boolean, prependTimestamp?: boolean): void {
 		const logFilePath = typeof _logFilePath === 'string' ?
 			_logFilePath :
 			(_logFilePath && this._logFilePathFromInit);
 
+		if (typeof prependTimestamp === 'boolean') {
```

Comment:
I did it this way (initializing to true in the else block) because of the reason below. Should I leave the timestamps in the beginning portion by default then?

Reply:
Using a default parameter would be equivalent to what you already have here, it won't affect what's happening in initialization.